test/test_checkers.py

```diff
@@ -99,6 +101,14 @@ class TestCheckerVersionParser:
             ("zlib", "libz.so.0", ["zlib"]),
         ],
     )
+    @classmethod
```

Comment:
You added this below `@pytest.mark.parametrize`.

You need to move it above line 43

Reply:
Thanks, I have moved it to line 43